docs/contributing.rst

```diff
@@ -182,145 +226,115 @@ Now that you have a general sense of how the repo is organized (and how you will
 
 .. image:: images/decorative_line4.png
   :width: 700px
   :height: 8px
   :align: center
-  :alt: decorative line
+  :alt: decorative line  
 
-4. When you start working on a new feature, **create/checkout a new branch** for that feature:
+5. Next, you will want to **render the handbook locally** so that you can get a sense of the impact of the changes you make on the website. 
 
 .. code-block:: bash
 
-    # create and switch to new branch called 'new-feature'
-    git checkout -b [new-feature]
-      e.g., git checkout -b testing
+    # render the handbook locally
+    make build
+
+* The output on your terminal will look something like the screenshot below, with the location of the html pages living in the location circled in blue. 
+
+.. image:: images/make_build_output.png
+  :width: 400px
+  :align: center
+  :alt: output you get after 'make build'
+  :class: with-border
+
+* Open up Finder and navigate to the directory :blue:`handbook/docs/_build/html`. Open :blue:`index.html`. This will open a local version of the handbook in your internet browser.
+
+.. important::
+
+    Use this opportunity to (a) view your changes and (b) make sure that your changes haven't messed up the rendering of the handbook!!
+
+* If you decide to make more modifications to the .rst file and want to view your changes, simply ``make build`` again in your terminal, then refresh the browser window that is displaying :blue:`index.html` and you should see your new modifications rendered.
 
 .. image:: images/decorative_line4.png
   :width: 700px
   :height: 8px
   :align: center
   :alt: decorative line
 
-5. Using your favorite source code text editor, **make changes to the the document you want to modify**. Use ``make build`` to (a) view your changes and (b) make sure that your changes haven't messed up the rendering of the handbook.
+6. When you are happy with your edits, you want to **add, commit, and push your changes to your feature branch** in your forked repo.
 
-* When you're making changes, it might be easy to lose track of what were the exact files that were modified. You can use ``git status`` to view the files that were changed in the working directory.
+* When you're making changes, it might be easy to lose track of which exact files were modified. You can use ``git status`` to view the files that were changed in the working directory.
 
 .. image:: images/screenshot_git_status.png
   :width: 500px
   :align: center
   :alt: screenshot of output for 'git status'
   :class: with-border
 
-.. image:: images/decorative_line4.png
-  :width: 700px
-  :height: 8px
-  :align: center
-  :alt: decorative line
-
-6. You want to **add, commit, and push your changes to your feature branch** in your forked repo.
-
 .. code-block:: bash
 
-    # add a modified file in the working directory to the staging area
-    git add docs/content_pages/01-01-howto.rst
+    # reminder yourself of which files have been modified
```

Comment:
"remind"

Reply:
Good catch, Sam! Lizzie, can you change this in your code? If you push the new code to your branch you don't have to submit a new pull request.

Reply:
donesies!